lisp/layouts/list_layout/layout.py

```diff
@@ -290,6 +291,11 @@ def double_clicked(self, event):
         if cue is not None:
             self.edit_cue(cue)
 
+    def select_event(self, event):
+        self._context_item = self.listView.itemAt(event.pos())
```

Comment:
Assigning the selected item to `self._context_item` it's not harmful but it shouldn't be done there.
Use a new local variable (eg: `item = ...`)